hyeminpark/hyems-workspace/campus-backend-project1/src/Exam1.java

```diff
@@ -0,0 +1,32 @@
+import java.util.Scanner;
+
+public class Exam1 {
+
+	public static void main(String[] args) {
+		// ������� ����Ѵ�.
+		// 1�ܺ��� 9�ܱ��� ����Ѵ�.
+		// Optional: ���� �� �� �Է¹޴´�.
+		//			  �� ��� 1~999������ �Է¸� �ȿ�ϰ� ����ó��
+		// Optional: ����Ͻ� ���� �и� �غ�����.
+		// 			 View �ܰ� ����Ͻ����� Ŭ������ ��� ���ؼ� �ڵ带 ���ο��� �и��غ�����.
+		
+		int mtp; // �� ��
+		int i;
+		Scanner sc = new Scanner(System.in);
+		
+		while(true) {
+			System.out.println("����� ������ �� �� �Է��ϼ��� : ");
+			mtp = sc.nextInt();
+			if (mtp > 999) {
```

Comment:
0 보다 작을 때 조건을 깜빡하신 것 같네요!
그래도 전체적으로 잘하셨네요! 
```suggestion
			if (mtp > 999 || mtp < 1) {
```